mantidimaging/gui/windows/operations/test/presenter_test.py

```diff
@@ -99,12 +105,16 @@ def test_apply_filter_to_all(self, apply_filter_mock: mock.Mock, _):
                          _wait_for_stack_choice=DEFAULT,
                          _do_apply_filter_sync=DEFAULT)
     def test_post_filter_success(self,
-                                 do_update_previews: Mock = Mock(),
-                                 _wait_for_stack_choice: Mock = Mock(),
-                                 _do_apply_filter_sync: Mock = Mock()):
+                                 do_update_previews: Optional[Mock] = None,
+                                 _wait_for_stack_choice: Optional[Mock] = None,
+                                 _do_apply_filter_sync: Optional[Mock] = None):
         """
         Tests when the operation has applied successfully.
         """
+        do_update_previews = _create_mock_if_none(do_update_previews)
+        _wait_for_stack_choice = _create_mock_if_none(_wait_for_stack_choice)
+        _do_apply_filter_sync = _create_mock_if_none(_do_apply_filter_sync)
+
```

Comment:
I think these might not be needed. https://docs.python.org/3/library/unittest.mock.html#patch-multiple looks like it will always create those mocks. So I think it could just change to:

```python
   def test_post_filter_success(self,
                                 do_update_previews: Mock,
                                 _wait_for_stack_choice: Mock,
                                 _do_apply_filter_sync: Mock):
```